src/MainWindow.vala

```diff
@@ -66,20 +62,19 @@ public class Music.MainWindow : Hdy.ApplicationWindow {
         now_playing.attach (headerbar, 0, 0);
         now_playing.attach (now_playing_view, 0, 1);
 
-        var now_playing_handle = new Hdy.WindowHandle ();
-        now_playing_handle.add (now_playing);
+        // var now_playing_handle = new Hdy.WindowHandle ();
```

Comment:
We have GtkWindowHandle now

Reply:
You probably don't need a window handle here at all. GtkHeaderBar and AdwHeaderBar are both draggable automatically.

Reply:
This is to make the entire background draggable so probably still want a handle here. Thanks!

Reply:
Ah ok, then you indeed need it. Just saw a few places with GtkHeaderBars within HdyWindowHandles in the past, so this looked like one of those places. :)